.github/SUPPORT.md

```diff
@@ -0,0 +1,60 @@
+# Support
+
+> This project has a [code of conduct][coc].
+> By interacting with this repository, organization, or community you agree to
+> abide by its terms.
+
+Hi!  👋
+We’re excited that you’re using **Flutter Boilerplate** and we’d love to help.
+To help us help you, please read through the following guidelines.
+
+Please understand that people involved with this project often do so for fun,
+next to their day job; you are not entitled to free customer service.
+
+## Contents
+
+*   [Questions](#questions)
+    *   [Asking quality questions](#asking-quality-questions)
+*   [Contributions](#contributions)
+
+## Questions
+
+Please chat and ask questions on [Discussions][chat]!
+Jump in there and lurk, talk to us, and help others.
+
+### Asking quality questions
+
+Help us help you!
+
+Spending time framing a question and adding support links or resources makes it
+much easier for us to help.
+It’s easy to fall into the trap of asking something too specific when you’re
+close to a problem.
+Then, those trying to help you out have to spend a lot of time asking additional
+questions to understand what you are hoping to achieve.
+
+Spending the extra time up front can help save everyone time in the long run.
+
+*   Try to define what you need help with:
+    *   Is there something in particular you want to do?
```

Comment:
Should be
* Is there something, in particular, you want to do?

Reply:
fixed